oss-internship-2020/gdal/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.10) 
+
+project(test CXX C)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(SAPI_ROOT "/usr/local/google/home/inach/work/sandboxed-api" CACHE PATH "Path to the Sandboxed API source tree")
+#   cmake .. -G Ninja -DSAPI_ROOT=$HOME/sapi_root
+
+set(SAPI_ENABLE_EXAMPLES OFF CACHE BOOL "")
+set(SAPI_ENABLE_TESTS OFF CACHE BOOL "")
+add_subdirectory("${SAPI_ROOT}"
+                 "${CMAKE_BINARY_DIR}/sandboxed-api-build"
+                 # Omit this to have the full Sandboxed API in IDE
+                 EXCLUDE_FROM_ALL)
+
+add_library(libgdal STATIC IMPORTED)
+set_property(TARGET libgdal PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/lib/libgdal.a")
+               
+find_package(PNG REQUIRED)
+
+target_link_libraries(libgdal INTERFACE
+    crypto expat jpeg 
+    /usr/lib/x86_64-linux-gnu/libproj.so
+    /usr/lib/x86_64-linux-gnu/libpcre.so
```

Comment:
This uses pre-installed packages, so we should try to find these. The easiest way to do this on Debian is to use pkg-config:
```
# Somewhere before line 21...
find_package(PkgConfig REQUIRED)
pkg_check_modules(libpcre REQUIRED IMPORTED_TARGET libpcre)
pkg_check_modules(libproj REQUIRED IMPORTED_TARGET libproj)
```

Then, in the `target_link_libraries`, replace the absolute paths with
```
PkgConfig::libpcre
PkgConfig::libproj
```